### ISSUE TITLE
Not found libs based on x86 in 1.5.10

### ISSUE THREAD

**Issue by nanguantong** [bytedeco/javacpp#751]
@saudet 
Hi, Not found libs based on x86 in 1.5.10. why?

```
Could not find javacpp-1.5.10-linux-x86.jar (org.bytedeco:javacpp:1.5.10).
Searched in the following locations:
    https://repo.maven.apache.org/maven2/org/bytedeco/javacpp/1.5.10/javacpp-1.5.10-linux-x86.jar

Could not find javacpp-1.5.10-windows-x86.jar (org.bytedeco:javacpp:1.5.10).
Searched in the following locations:
    https://repo.maven.apache.org/maven2/org/bytedeco/javacpp/1.5.10/javacpp-1.5.10-windows-x86.jar
```

**Comment by saudet** [comment]
I have not heard of any users needing those for years, but if you need then, please let me know

**Comment by nanguantong** [comment]
Yeah, not commonly used, only missing x86 libs updated from 1.5.8 to 1.5.10 in our project config, so why not keeping it as it is.

**Comment by nanguantong** [comment]
@saudet 

Another problem that `version `GLIBC_2.28' not found (required by /xxx/.javacpp/cache/ffmpeg-6.1.1-1.5.10-linux-x86_64.jar/org/bytedeco/ffmpeg/linux-x86_64/libavutil.so.58)` on ubuntu-18.04 with glibc 2.27.

Could you please build the libs based on glibc 2.27 ?

**Comment by saudet** [comment]
Again, if enough people need binaries for old version of Linux, that's something that will happen, but that's not the case. Not many people need binaries for old version of Linux.
